playbooks/common/openshift-node/config.yml

```diff
@@ -79,7 +80,8 @@
     with_items: nodes_needing_certs
 
 - name: Configure node instances
-  hosts: oo_nodes_to_config
+  hosts:
+    - oo_nodes_to_config:oo_infra_to_config
```

Comment:
Although, now that I look at it, I don't see oo_infra_to_config used anywhere else. Is it even being populated?